packages/block-editor/src/components/writing-flow/use-tab-nav.js

```diff
@@ -10,11 +10,11 @@ import { useRef } from '@wordpress/element';
 /**
  * Internal dependencies
  */
+import { isInSameBlock } from '../../utils/dom';
 import { store as blockEditorStore } from '../../store';
 
 export default function useTabNav() {
 	const container = useRef();
-	const focusCaptureBeforeRef = useRef();
```

Comment:
Since block toolbar is no longer receiving focus via Shift+Tab, remove the before.